File: even_numbers/even_numbers.py
import sys

def main(file_arg):
    with open(file_arg) as f:
        for line in f:
            number = int(line)
            if number % 2 == 0:
                print('1')
            else:
                print('0')



if __name__ == '__main__':
    main(sys.argv[1])

File: even_numbers/even_numbers.cpp
#include <iostream>
#include <sstream>
#include <fstream>

int main(int argc, char *argv[])
{
    using namespace std;

	ifstream file(argv[1]);
	int number;
	string line;

	while(getline(file, line))
	{
	    istringstream iss(line);
		iss >> number;

		if (number % 2 == 0) { cout << "1" << endl;}
		else { cout << "0" << endl;}
	}

	return 0;
}
